.gitignore

```diff
@@ -0,0 +1,2 @@
+.idea
```

Comment:
please take a look at this site  https://www.toptal.com/developers/gitignore and create more specific ignore file

Reply:
@artemmoroz done, please review